### PR TITLE
Addition of us-east-2 to the inventory ini

### LINKED ISSUE #143: AWS: failed with 'dict object' has no attribute 'tag_openshift_role_master'
TASK [instance-groups : Add masters to requisite groups] ***********************
task path: /root/AWS/openshift-ansible-contrib/reference-architecture/aws-ansible/playbooks/roles/instance-groups/tasks/main.yaml:9
fatal: [localhost]: FAILED! => {
    "failed": true
}

MSG:

'dict object' has no attribute 'tag_openshift_role_master'

### PR DESCRIPTION
**Pull request by cooktheryan**
Resolves #143

### CODE REVIEW

**Comment by cooktheryan** [comment]
atomic-ci-test

File: reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # AWS regions to make calls to. Set this to 'all' to make request to all regions
 # in AWS and merge the results together. Alternatively, set this to a comma
 # separated list of regions. E.g. 'us-east-1,us-west-1,us-west-2'
-regions = us-east-1,us-west-1,us-west-2,sa-east-1,eu-west-1,eu-central-1,ap-northeast-1,ap-southeast-1,ap-southeast-1,ap-southeast-2
+regions = us-east-1,us-east-2,us-west-1,us-west-2,sa-east-1,eu-west-1,eu-central-1,ap-northeast-1,ap-southeast-1,ap-southeast-1,ap-southeast-2
```
Comment (detiber) [inline reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini]: Would it make more sense to just set this to 'all' instead?

Comment (cooktheryan) [inline reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini]: it definitely would

**Comment by cooktheryan** [comment]
aos-ci-test

**Comment by cooktheryan** [comment]
aos-ci-test

**Pull request merged**